generators/server/templates/src/main/java/package/config/_FeignConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package <%=packageName%>.config;
+
+import org.springframework.cloud.netflix.feign.EnableFeignClients;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+
+@Configuration
+@Profile("!test")
+@EnableFeignClients(basePackages = "<%=packageName%>.client")
+public class FeignConfiguration {
```

Comment:
Shouldn't this be put directly on the main application instead of pulling a configuration class ?